backend/api/go_client/auth.pb.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2021 Google LLC
```

Comment:
Hmm, looks like we have double license headers

Reply:
will fix this later when we have the generation working